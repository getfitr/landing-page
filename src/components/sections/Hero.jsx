
const Hero = () => (
  <div>HERO</div>
);

export default Hero;
