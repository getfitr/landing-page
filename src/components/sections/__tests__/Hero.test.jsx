import renderer from 'react-test-renderer';
import Hero from '../Hero';

test('Show the hero component', () => {
  const hero = renderer.create(<Hero />);
  const tree = hero.toJSON();
  expect(tree).toMatchSnapshot();
});