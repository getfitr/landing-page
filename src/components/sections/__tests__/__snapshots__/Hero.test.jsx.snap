// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show the hero component 1`] = `
<div>
  HERO
</div>
`;
